fireblocks_sdk/sdk.py

```diff
@@ -615,16 +615,11 @@ def set_address_description(
             tag (string, optional): The XRP tag, or EOS memo, for which to set the description
             description (string, optional): The description to set, or none for no description
         """
-        if tag:
-            return self._put_request(
-                f"/v1/vault/accounts/{vault_account_id}/{asset_id}/addresses/{address}:{tag}",
-                {"description": description or ""},
-            )
-        else:
-            return self._put_request(
-                f"/v1/vault/accounts/{vault_account_id}/{asset_id}/addresses/{address}",
-                {"description": description or ""},
-            )
+        target = f'{address}:{tag}' if tag else address
+        return self._put_request(
+            f"/v1/vault/accounts/{vault_account_id}/{asset_id}/addresses/{target}",
+            {"description": description or ""},
+        )
```

Comment:
thank you for this! it looks better.
we decided not to merge for now since its just optimization.